src/bot.py

```diff
@@ -192,15 +196,50 @@ def is_last_run(self):
                 found_unfinished_run = True
                 break
         return not found_unfinished_run
+    
+    def _rebuild_as_asset_to_trigger(trigger_to_assets: dict):
+        result = {}
+        for key in trigger_to_assets.keys():
+            for asset in trigger_to_assets[key]:
+                result[asset] = key
+        return result
+
+    def on_init(self):
+        keyboard.release(self._config.char["stand_still"])
+        transition_to_screens = Bot._rebuild_as_asset_to_trigger({
+            "select_character": [
+                "MAIN_MENU_TOP_LEFT","MAIN_MENU_TOP_LEFT_DARK"
+            ],
+            "start_from_town": [
```

Comment:
We have now 2 places where we have to take care of the list of "town templates". Maybe provide a getter in the town manager to get they asset names and use the getter here.